### ISSUE TITLE
Add support for factory image overlays

### ISSUE THREAD

**Issue by solidDoWant** [budimanjojo/talhelper#676]
Hi folks. As discussed [in Discord here](https://discord.com/channels/673534664354430999/1095849307183255593/1300658492021018694), it'd be really nice if talhelper supported separate overlays for ISOs and installer images.

The primary use case is this: when installing Talos, I need to set cmdline parameters for the very first boot so that the Talos ISO image can talk with the network. Examples of these parameters include `ip=1.2.3.4...` and `vlan=vlan.100:eno1`.

However, once bootstrapping is complete and the "installer" image is installed, these parameters are no longer needed or desired. Retaining these parameters can introduce several problems:
* Dracut (I think it's dracut, might be something else) will always wait 120 seconds for the network, even if it is available, which prolongs boot times.
* IP addresses, vlan interfaces, routes, etc. will still be configured this way will still exist regardless of the `network-interfaces` settings. If I later want to change something set here, I have to upgrade the image rather than just deploy a new config. In some corner cases, a change can screw up the network config if there are conflicting values in both the config `network-interfaces` field, and the cmdline.

Ideally the implementation would involve adding an "overlay" field of some kind for both the ISO and installer images. This would allow for a baseline, "common" config, as well as image-specific changes.

**Comment by budimanjojo** [comment]
The easiest way to achieve this is to add something like `isoSchematicOverride`.
It will only be used for `genurl iso` if defined, `genurl iso` will still use the current `schematic` field if not defined. Is this a reasonable fix for you?

**Comment by solidDoWant** [comment]
Yea that'd be great

**Comment by budimanjojo** [comment]
PR created, `isoSchematic` instead of `isoSchematicOverride`. Feel free to share your thoughts, it's a super simple addition and I think this should be all you need. Thank you!

**Comment by budimanjojo** [comment]
FYI v3.0.9 released with this new feature, thank you!
